wfs3/collection_meta_data.go

```diff
@@ -93,8 +93,8 @@ func CollectionMetaData(name string, p *data_provider.Provider, serveAddress str
 		return nil, "", fmt.Errorf("Invalid collection name: %v", name)
 	}
 
-	collectionUrl := fmt.Sprintf("http://%v/collections/%v", serveAddress, name)
-	cInfo := CollectionInfo{Name: name, Links: []*Link{{Rel: "self", Href: collectionUrl}}}
+	collectionUrl := fmt.Sprintf("%v/collections/%v", serveAddress, name)
```

Comment:
Per previous comment (go-wfs-config.toml) I don't think the protocol belongs in serveAddress.